test/integration/confirmations/signatures/personalSign.test.tsx

```diff
@@ -163,9 +163,7 @@ describe('PersonalSign Confirmation', () => {
 
     expect(getByText('Signature request')).toBeInTheDocument();
     expect(
-      getByText(
-        'Only confirm this message if you approve the content and trust the requesting site.',
-      ),
+      getByText('Review request details before you confirm.'),
```

Comment:
[Optional] We can leverage `tEn('i18nKey')` in `test/lib/i18n-helpers.js` for these tests.

Reply:
awesome! didn't know about this helper. To not block merge, I can look into adding this in a different PR